core/linux-dist/evdev.cpp

```diff
@@ -118,40 +118,48 @@
 			strstr(keycode.c_str(), "BTN_") != NULL ||
 			strstr(keycode.c_str(), "ABS_") != NULL)
 		{
-			if(libevdev_available)
+			if (libevdev_available)
 			{
 				int type = ((strstr(keycode.c_str(), "ABS_") != NULL) ? EV_ABS : EV_KEY);
 				code = libevdev_event_code_from_name(type, keycode.c_str());
 			}
-			if(code < 0)
+			
+			if (code < 0)
 			{
 				printf("evdev: failed to find keycode for '%s'\n", keycode.c_str());
 			}
 			else
 			{
 				printf("%s = %s (%d)\n", dc_key.c_str(), keycode.c_str(), code);
 			}
-			return code;
 		}
-
-		code = cfg->get_int(section, dc_key, -1);
-		if(code >= 0)
+		else
 		{
-			char* name = NULL;
-			if(libevdev_available)
-			{
-				int type = ((strstr(dc_key.c_str(), "axis_") != NULL) ? EV_ABS : EV_KEY);
-				name = (char*)libevdev_event_code_get_name(type, code);
-			}
-			if (name != NULL)
-			{
-				printf("%s = %s (%d)\n", dc_key.c_str(), name, code);
-			}
-			else
+			code = cfg->get_int(section, dc_key, -1);
+			if(code >= 0)
```

Comment:
https://github.com/reicast/reicast-emulator/pull/1282/commits/369ecf2db2f33eb54f2ca8c36c18e932ce06bac9#diff-6b992ef7d6b919c838ffea6c7a09e897R127

Reply:
?

Reply:
One of these things is not like the others.

Reply:
`if (code < 0)` (127)
`if(code >= 0)` (139)

Reply:
Ah, ok. I'll fix that.
But it won't be in the next few days.